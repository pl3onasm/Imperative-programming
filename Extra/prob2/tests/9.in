954527775465122004546937346598205367987885031625986260532659784848484847956868359923643461905122456566132326554250141505254205450552421541648538789764822596858335656441255285685256464452858204541156443358582525445572725864455475274361412289765389564157484956835292031623542101004253586689146262645343242126006060114163352536451642162352762567246246576564675977872837303505380530070524720127892536879202307324214656574275654771407017615075641575210527815281562715671201472604526705237105276052370273021734017648
4556144655389567345166222265383386595678374210050553260032311425050432365365898976654653342420102005055246216323385686868686451174747772253293336936336939656545415454656166465653664664644665352852246451662653839639668685997646538838338324121205164797686834242133062595658635952151564166341634374324152056506623535354220442313526535322342625543583358342442122412102202124342732515419647879787987875454612151541549645455555550002655405459640515614596
