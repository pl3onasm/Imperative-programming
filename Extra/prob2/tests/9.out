954527775465122004546937346598205367987885031625986260532659789404629503346435705089865727288509052244506536604803401537565630500984786907547515444418165016960340711687471609070943333139309379288928696651919461782512382409870929930527877943430054208822837809081743694277381740672939584335685100983667363331170857800997594669514237820988714718810412918199050302024893810128915884290967033654427859221827850907572999016398066873609141957808156994858407603269438170283353014154172160792655278707775732662249632244
